docs/hierarchical-pipeline.md

```
@@ -0,0 +1,142 @@
+}
+```
+
+Every key understood by the flat plugin is also understood here.
+
+---
+
+## Two-stage routing
+
+Each registered intent label is of the form `skill_id:intent_name`. The plugin treats the `skill_id` prefix as the **domain**.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)